rtpcodec.go

```diff
@@ -155,6 +156,27 @@ func findRTXPayloadType(needle PayloadType, haystack []RTPCodecParameters) Paylo
 	return PayloadType(0)
 }
 
+// Given RTX CodecParameters find the primary CodecParameters if one exists.
+func findPrimaryPayloadTypeForRTX(needle RTPCodecParameters, haystack []RTPCodecParameters) PayloadType {
+	parts := strings.Split(needle.SDPFmtpLine, "=")
```

Comment:
It might be worth adding an additional sanity check here that the mime type for `needle` is actually the RTX mime type - just in case someone calls this with a non RTX codec.

Reply:
Or maybe `(PayloadType, bool)` with the second return value indicating if needle is `rtx`?